vapoursynth/src/map/mod.rs

```diff
@@ -190,10 +189,10 @@ fn handle_get_prop_error(error: i32) -> Result<()> {
         Ok(())
     } else {
         Err(match error {
-            x if x == ffi::VSGetPropErrors::peUnset as i32 => Error::KeyNotFound,
-            x if x == ffi::VSGetPropErrors::peType as i32 => Error::WrongValueType,
-            x if x == ffi::VSGetPropErrors::peIndex as i32 => Error::IndexOutOfBounds,
-            _ => unreachable!(),
+            x if x == ffi::VSMapPropertyError_peUnset as i32 => Error::KeyNotFound,
+            x if x == ffi::VSMapPropertyError_peType as i32 => Error::WrongValueType,
+            x if x == ffi::VSMapPropertyError_peIndex as i32 => Error::IndexOutOfBounds,
+            _ => Error::UnknownError,
```

Comment:
Can this happen?

Reply:
I suppose it's to avoid panicking in the lib but it doesn't seem to be possible.
There is a generic `peError` type but not sure when it can happen.

Reply:
It seems there is a new `VSMapPropertyError_peError` code which represents a generic error, so this would refer to that case. But since the error codes are all integers, we also need a branch for the extra integers anyway.

The alternative would be a specific case for `VSMapPropertyError_peError`, and make a new branch `unreachable!()` for unexpected error codes.